_posts/2020-09-20-processing-files-using-java-8-streams.md

```diff
@@ -0,0 +1,382 @@
+﻿---
+title: "Processing Files with Java 8 Streams"
+categories: [java]
+date: 2020-09-20 06:00:00 +1000
+modified: 2020-09-20 06:00:00 +1000
+author: default
```

Comment:
I have added this header. Please add yourself to `data/authors.yml` and put your author key in this line so that you are mentioned as the author of the article. 

I'll take care of the rest of the header.

Reply:
Done